### DISCUSSION TITLE
Armored Core: Master of Arena Mouse Injector Support

### DISCUSSION THREAD

**Discussion by GoldenLumia** [garungorp/MouseInjectorDolphinDuck#22] [Show and tell]
My goal is to have mouse injector support for all Armored Core PS1 and PS2 games, but obviously I can't promise that. 

Either way, I hope you enjoy: https://github.com/GoldenLumia/MouseInjectorDolphinDuck/releases/tag/v17

And thank you again garungorp for the tutorial video.

**Comment by lagoooona** [comment]
Amazing! Thank you so much for this!

**Comment by KingLich1** [comment]
thanks

**Comment by UserGit-2021** [comment]
> My goal is to have mouse injector support for all Armored Core PS1 and PS2 games, but obviously I can't promise that.
> 
> Either way, I hope you enjoy: https://github.com/GoldenLumia/MouseInjectorDolphinDuck/releases/tag/v17
> 
> And thank you again garungorp for the tutorial video.

Are you planning support for Medal of Honor PS1, Tecmo Deseption and TimeSplitters 2?

**Reply by vl4sty** [reply]
I looked around the unfinished MoH hack for PS1. I found the playserbase pointer, so now the aiming works in every level. I will still need to do a lot work on the turrets and precise aim, plus I will need to look for a function/variable that enables the crosshair, so it always stays on. I do not know how well it will work in the end, but I will try. I will finnish it when I have some free time, i.e. definetly in July lol

**Reply by UserGit-2021** [reply]
> I looked around the unfinished MoH hack for PS1. I found the playserbase pointer, so now the aiming works in every level. I will still need to do a lot work on the turrets and precise aim, plus I will need to look for a function/variable that enables the crosshair, so it always stays on. I do not know how well it will work in the end, but I will try. I will finnish it when I have some free time, i.e. definetly in July lol

Thank you, I'll wait until July, it's not a problem. Good luck!

**Comment by GoldenLumia** [comment]
Hey there, if anyone is interested in testing AC2: Another Age, I've added preliminary support to [my branch](https://github.com/GoldenLumia/MouseInjectorDolphinDuck/releases/tag/v19-prerelease2) along with fixing the lingering MoA issues.
